### PR TITLE
Categories functionality implemented, filtering not yet possible

### PR DESCRIPTION
**Pull request by Marcholio**

### CODE REVIEW

File: app/Assets/style.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@ $cancel: #E86753;
 
   #create-form {
     margin: 16px 0 0 0;
+    padding-left: 5vw;
```
Comment (marinp1) [inline app/Assets/style.scss:26]: Is padding relative to viewport width good idea?

File: app/TransactionView.js [hunk 2]
```diff
@@ -255,3 +279,19 @@ class Confirm extends React.Component {
...
+class CategorySelect extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <select className='categoryCol' value={ this.props.category } onChange={ this.props.valueChange } name='category'>
+        { _.map(this.props.categories, category =>
+          <option key={ category.name } value={ category.name }>{ category.name }</option>
```
Comment (marinp1) [inline app/TransactionView.js]: Should the mapping values be the category's id instead of name?

Comment (Marcholio) [reply app/TransactionView.js]: ids are numbers and therefore won't mean anything to user

File: app/Assets/style.scss (hunk 1 above)
Comment (Marcholio) [reply app/Assets/style.scss:26]: Looks good when tested on two different screens

File: app/TransactionView.js (hunk 2 above)
Comment (marinp1) [reply app/TransactionView.js]: I meant the key-attribute, can it contain special characters?

Comment (Marcholio) [reply app/TransactionView.js]: point

File: app/Assets/style.scss (hunk 1 above)
Comment (marinp1) [reply app/Assets/style.scss:26]: Ok

**Pull request merged**